zrml/asset-router/src/tests/destroy.rs

```diff
@@ -21,12 +21,12 @@ use super::*;
 use frame_support::traits::tokens::fungibles::Inspect;
 
 fn test_helper(asset: Assets, initial_amount: <Runtime as crate::Config>::Balance) {
-    assert!(AssetRouter::asset_exists(asset));
     assert_ok!(<AssetRouter as orml_traits::MultiCurrency<AccountId>>::deposit(
         asset,
         &ALICE,
         initial_amount
     ));
+    assert!(AssetRouter::asset_exists(asset));
```

Comment:
What's the purpose of this change?

Reply:
See https://github.com/zeitgeistpm/zeitgeist/pull/1256#discussion_r1497273125